app/connector/mongodb/src/main/resources/META-INF/syndesis/connector/mongodb3.json

```diff
@@ -183,14 +210,30 @@
                 "required": true,
                 "secret": false,
                 "type": "string"
+              },
+              "filter": {
+                "deprecated": false,
+                "displayName": "Update expression",
+                "group": "common",
+                "javaType": "java.lang.String",
+                "kind": "path",
+                "labelHint": "Expression used as a criteria to update a set of documents. Use :#variable to set variables eventually mapped to any incoming input.",
+                "placeholder": "{\"color\": :#color}",
+                "order": "2",
+                "required": true,
```

Comment:
If we're adding a required property, we probably need a migration to set `configuredProperty` to integration's existing action steps.

Reply:
Do we have some document to explain how to provide that migration?

Reply:
There's a bit here: https://github.com/syndesisio/syndesis/blob/master/project/proposals/971-upgrade.adoc#step-upgrade-db have a look at the scripts [here](https://github.com/syndesisio/syndesis/tree/master/app/server/runtime/src/main/resources/migrations) for ideas on how this can be done, and make sure to upgrade the schema version [here](https://github.com/syndesisio/syndesis/blob/master/app/common/model/src/main/java/io/syndesis/common/model/Schema.java).

Reply:
I don't think that is really needed. All the actions provided in 7.6 will be using a new connector ID. Migrating 7.5 to 7.6 is not compatible as there would be missing mandatory criteria and data mapping, so, if any migration is needed it must be performed through a kick off of a new integration with the new parameters required.